modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics
+. In the web console, go to Operators → Installed Operators.
+. Under the Provided APIs heading for the NetObserv Operator, select Flow Collector.
+. Select cluster then select the YAML tab.
+. Edit the `FlowCollector`  to add the following parameters for `spec.processor`:
+  `processor.subnetLabels` field to create the FlowMetrics API.
++
+[source,yaml]
+—-
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+Spec:
+# …
+processor:
+  addZone: true                <1>
+  subnetLabels:
+    openShiftAutoDetect: true  <2>
+    customLabels:
+    - cidrs: []
+      name: ""
+# …
+—-
+<1> Optional: Configure the ability to see zones in the metrics data. 
+<2> spec.processor.subnetLabels.openShiftAutoDetect` set to `true` to enable custom metrics. 
```

Comment:
You can remove this part as we finally decided to enable this by default

Reply:
Removed